docs/daily-usage.md

```diff
@@ -93,10 +93,16 @@ Start Odoo with:
 invoke start
 ```
 
+Apply some common configurations for devel to work fine with:
+
+```bash
+invoke preparedb
+```
+
 All of the above in one shot:
 
 ```bash
-invoke img-pull img-build --pull git-aggregate resetdb start
+invoke img-pull img-build --pull git-aggregate resetdb preparedb start
```

Comment:
This is not needed, as preparedb is executed by default when resetting the DB, unless you specify `--no-populate`
```suggestion
invoke img-pull img-build --pull git-aggregate resetdb start
```
And, as we are changing :smile: , this is the default also:
```suggestion
invoke img-pull img-build git-aggregate resetdb start
```